components/tabs/stories/tabs.test.js

```diff
@@ -90,7 +107,27 @@ export const TabsGroups = Variants({
 		},
 		{
 			testHeading: "Icon only",
-			iconOnly: true
+			iconOnly: true,
 		},
+		{
+			testHeading: "Selected tab focused",
```

Comment:
Oh yeah, I wanted to ask- this is purposeful to have the horizontal focus be slightly different from the vertical focus, correct? The indicator is outside the ring on vertical, while it's inside the focus ring on horizontal focus. I don't think I have an issue with the visual difference seeing them in the test grid- it just caught my eye when I was testing it outside of the test grid.

<img width="131" height="91" alt="Screenshot 2025-07-16 at 9 03 55 AM" src="https://github.com/user-attachments/assets/a12b16e3-2e26-4cf0-888e-1f3bd2935813" />
<img width="116" height="92" alt="Screenshot 2025-07-16 at 9 03 48 AM" src="https://github.com/user-attachments/assets/a9a3fa50-ec37-4530-8473-f65aad830962" />